index.js

```diff
@@ -1 +1,63 @@
 // implement your API here
+// import express from 'express' //es2015 Module Import
+const express = require('express');
+
+const db = require('./data/db.js')
+
+const server = express();
+
+server.use(express.json()); //tells express how to write json
+// does not know how to do out of the box
+
+    server.get('/', (req, res) => {
+            res.send("It's alive!");
+        })
+
+    server.post('/api/users', (req, res) => {
+
+    const userInfo = req.body;
+    console.log('requst body', userInfo);
+    
+    db.insert(users)
```

Comment:
we wanna pass in the req.body here, `users` is nothin! but `userInfo` is (or just straight up req.body)